package-lock.json

```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "us-bank-account-validator",
-      "version": "0.2.0",
+      "version": "1.0.0",
```

Comment:
Should this be a minor version bump? I'm only seeing minor and patch updates to any dependency. Is there a breaking change involved in this PR?